python/ribasim_testmodels/ribasim_testmodels/continuous_control.py

```diff
@@ -25,7 +25,9 @@ def outlet_continuous_control_model() -> Model:
         Node(1, Point(0, 0)),
         [
             level_boundary.Time(
-                time=pd.date_range(start="2020-01-01", end="2021-01-01", periods=100),
+                time=pd.date_range(
+                    start="2020-01-01", end="2021-01-01", periods=100, unit="ms"
+                ),
```

Comment:
This needed `unit="ms"` otherwise the nanosecond precision times get a ValidationError now.

Reply:
Can't we automatically convert such timestamps? I fear that users might hit this and have to find this fix manually.

Reply:
The only point where we got nanosecond precision was with the `data_range` with periods pattern. I feel like this pattern will rarely be used for real models, maybe only synthetic ones like our docs. I don't think I've ever come across hydrological data with nanosecond precision.

Nanosecond units will automatically convert to millisecond as long as it doesn't lose data. The ValidationError is also quite good. I prefer not to throw a warning on this as it usually doesn't matter. If you know an easy way to round or truncate to the nearest millisecond I'd be ok with that as well. But I doubt many users will run into this.